From dd3f222e3b03ce60ac2c48ad7f7aa15013a945bf Mon Sep 17 00:00:00 2001
From: Jerry Hung <jerry.hung@gumstix>
Date: Wed, 8 Feb 2017 13:51:09 -0800
Subject: [PATCH 10/13] Overo : Updating dts for Morty/4.8

Overo is not able to boot because the mmc is not powered.

Changing vmmc-supply setting makes the boot successfully

Signed-off-by: Jerry Hung <jerry@gumstix.com>
---
 arch/arm/boot/dts/omap3-overo-base.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/omap3-overo-base.dtsi b/arch/arm/boot/dts/omap3-overo-base.dtsi
index bd22cdb..9322a2f 100644
--- a/arch/arm/boot/dts/omap3-overo-base.dtsi
+++ b/arch/arm/boot/dts/omap3-overo-base.dtsi
@@ -185,7 +185,6 @@
         vin-supply = <&vbat>;
         gpio = <&gpio6 4 GPIO_ACTIVE_HIGH>;	/* gpio_164: BT Enable */
         startup-delay-us = <70000>;
-        enable-active-high;
 };
 
 /* optional on board WiFi */
@@ -194,8 +193,9 @@
 	pinctrl-names = "default";
 	pinctrl-0 = <&mmc2_pins>;
 	ti,bus-width = <4>;
-	vmmc-supply = <&wlcore_wl_en>;
-	vmmc_aux-supply = <&wlcore_bt_en>;
+	vmmc-supply = <&vmmc2>;
+	vmmc_aux-supply = <&wlcore_wl_en>;
+	vqmmc-supply = <&wlcore_bt_en>;
 	ti,non-removable;
 	ti,needs-special-hs-handling;
 	cap-power-off-card;
-- 
2.7.4

